app/docs/md/learn/starter-project/configuration.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Configuration
+---
+
+Enhance starter projects include several tools to get you up and running with best practices as easily as possible (like the Enhance Styles library and the Enhance Image component and transformation service). While these tools come with sensible defaults baked in, each can also be customized to best suit your project’s requirements.
+
+To customize these (and other tools in the future), we expose a single configuration file (`enhance.json`) at the root of each project.
```

Comment:
Undecided: should we include `enhance.json` with the default configs since Styles and Image fall back to defaults on their own? Would be good to surface the file, but it's also not strictly necessary.

Reply:
Further: if `enhance.json` doesn't exist, how will Enhance get the default configs to include in `store.enhance`? 🤔